.golangci.yaml

```diff
@@ -8,7 +8,7 @@ linters:
     - durationcheck
     - errname
     - errorlint
-    - exportloopref
+    - copyloopvar
```

Comment:
we tried to keep them in ascii ordered for finding them later easily, mind changing this?

Reply:
done